keras_cv/models/object_detection/yolo_v8/yolo_v8.py

```diff
@@ -0,0 +1,604 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import copy
+
+import tensorflow as tf
+from keras import layers
+from tensorflow import keras
+
+import keras_cv
+from keras_cv import bounding_box
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.backbone_presets import backbone_presets
+from keras_cv.models.backbones.backbone_presets import (
+    backbone_presets_with_weights,
+)
+from keras_cv.models.object_detection import predict_utils
+from keras_cv.models.object_detection.__internal__ import unpack_input
+from keras_cv.models.object_detection.retina_net import RetinaNetLabelEncoder
+from keras_cv.models.object_detection.yolo_v8.compat_anchor_generation import (
+    get_anchors,
+)
+from keras_cv.models.object_detection.yolo_v8.yolo_v8_presets import (
+    yolo_v8_backbone_presets,
+)
+from keras_cv.models.object_detection.yolo_v8.yolo_v8_presets import (
+    yolo_v8_backbone_presets_with_weights,
+)
+from keras_cv.models.object_detection.yolo_v8.yolo_v8_presets import (
+    yolo_v8_presets,
+)
+from keras_cv.models.task import Task
+from keras_cv.utils.python_utils import classproperty
+from keras_cv.utils.train import get_feature_extractor
+
+BATCH_NORM_EPSILON = 1e-3
+BATCH_NORM_MOMENTUM = 0.97
+
+
+def conv_bn(
```

Comment:
I wonder if we can give this layer more descriptive name like `stack` or `block` a la resnet

Reply:
Done -- I've renamed these to use the backbone naming convention of `apply_xyz`